### PR TITLE
en, zh: update doc for backup and restore

### PR DESCRIPTION
**Pull request by DanielZhangQD**
### What is changed, added or deleted? (Required)
update doc for backup and restore

### Which TiDB Operator version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v1.1 (TiDB Operator 1.1 versions)
- [ ] v1.0 (TiDB Operator 1.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from: 
- Other reference link(s):

### CODE REVIEW

File: en/backup-to-s3.md [hunk 1]
```diff
@@ -157,15 +173,16 @@ Refer to [Ad-hoc full backup prerequisites](backup-to-aws-s3-using-br.md#prerequ
...
+        region: ${region}
+        bucket: ${bucket}
         # storageClass: STANDARD_IA
         # acl: private
         # endpoint:
     storageClassName: local-storage
     storageSize: 10Gi
     ```
 
-In the above two examples, all data of the TiDB cluster is exported and backed up to Amazon S3 and Ceph respectively. You can ignore the `region`, `acl`, `endpoint`, and `storageClass` configuration items in the Amazon S3 configuration. S3-compatible storage types other than Amazon S3 can also use configuration similar to that of Amazon S3. You can also leave the configuration item fields empty if you do not need to configure these items as shown in the above Ceph configuration.
```
Comment (shuijing198799) [inline en/backup-to-s3.md:168]: I think ceph only need to fill endpoint

Comment (DanielZhangQD) [reply en/backup-to-s3.md:168]: Ceph also supports acl, storageClass, etc.

File: en/backup-to-s3.md [hunk 2]
```diff
@@ -27,6 +27,19 @@ Refer to [Ad-hoc full backup prerequisites](backup-to-aws-s3-using-br.md#prerequ
 
 ### Ad-hoc backup process
 
+> **Note:**
+>
+> * If need to add `Prefix` to the backup data path, you can add the `Prefix` in the `bucket` configuration, e.g. `bucket: demo-bucket/demo-prefix`
```
Comment (ran-huang) [inline en/backup-to-s3.md:32]:
```suggestion
> * If you need to add `Prefix` to the backup data path, configure the `Prefix` in the `bucket` configuration in the YAML file, e.g.: `bucket: demo-bucket/demo-prefix`
```

File: en/backup-to-s3.md [hunk 3]
```diff
@@ -27,6 +27,19 @@ Refer to [Ad-hoc full backup prerequisites](backup-to-aws-s3-using-br.md#prerequ
...
+> * Due to the [issue](https://github.com/rclone/rclone/issues/1824) of `rclone`, if the backup data is stored in AWS S3 and the `AWS-KMS` encryption is enabled, you have to configure `spec.s3.options` as below in the yamls used in the following procedures, for more detail, refer to the [document](https://rclone.org/s3/#key-management-system-kms):
```
Comment (ran-huang) [inline en/backup-to-s3.md:33]:
```suggestion
> * Due to the [issue](https://github.com/rclone/rclone/issues/1824) of `rclone`, if the backup data is stored in AWS S3 and the `AWS-KMS` encryption is enabled, you need to configure `spec.s3.options` in the YAML file in the following steps:
```

File: en/backup-to-s3.md [hunk 4]
```diff
@@ -228,6 +247,21 @@ The prerequisites for the scheduled backup is the same as the [prerequisites for
 
 ### Scheduled backup process
 
+> **Note:**
+>
+> * If need to add `Prefix` to the backup data path, you can add the `Prefix` in the `bucket` configuration, e.g. `bucket: demo-bucket/demo-prefix`
```
Comment (ran-huang) [inline en/backup-to-s3.md:252]:
```suggestion
> * If you need to add `Prefix` to the backup data path, configure the `Prefix` in the `bucket` configuration in the YAML file, e.g.: `bucket: demo-bucket/demo-prefix`
```

File: en/backup-to-s3.md [hunk 5]
```diff
@@ -228,6 +247,21 @@ The prerequisites for the scheduled backup is the same as the [prerequisites for
...
+> * Due to the [issue](https://github.com/rclone/rclone/issues/1824) of `rclone`, if the backup data is stored in AWS S3 and the `AWS-KMS` encryption is enabled, you have to configure `spec.backupTemplate.s3.options` as below in the yamls used in the following procedures, for more detail, refer to the [document](https://rclone.org/s3/#key-management-system-kms):
```
Comment (ran-huang) [inline en/backup-to-s3.md:253]:
```suggestion
> * Due to the [issue](https://github.com/rclone/rclone/issues/1824) of `rclone`, if the backup data is stored in AWS S3 and the `AWS-KMS` encryption is enabled, you need to configure `spec.backupTemplate.s3.options` in the YAML file in the following steps:
```

File: en/backup-to-s3.md [hunk 6]
```diff
@@ -27,6 +27,19 @@ Refer to [Ad-hoc full backup prerequisites](backup-to-aws-s3-using-br.md#prerequ
...
+> * Due to the [issue](https://github.com/rclone/rclone/issues/1824) of `rclone`, if the backup data is stored in AWS S3 and the `AWS-KMS` encryption is enabled, you have to configure `spec.s3.options` as below in the yamls used in the following procedures, for more detail, refer to the [document](https://rclone.org/s3/#key-management-system-kms):
+>
+>     ```yaml
+>     spec:
+>       ...
+>       s3:
+>         ...
+>         options:
+>         - --ignore-checksum
+
```
Comment (ran-huang) [inline en/backup-to-s3.md:42]:
```suggestion
>     ```
>
> * For more details, refer to the [rclone | Key Management System](https://rclone.org/s3/#key-management-system-kms).

```

File: en/backup-to-s3.md [hunk 7]
```diff
@@ -228,6 +247,21 @@ The prerequisites for the scheduled backup is the same as the [prerequisites for
...
+>     ```yaml
+>     spec:
+>       ...
+>       backupTemplate:
+>         ...
+>         s3:
+>           ...
+>           options:
+>           - --ignore-checksum
+
```
Comment (ran-huang) [inline en/backup-to-s3.md:264]:
```suggestion
>     ```
>
> * For more details, refer to the [rclone | Key Management System](https://rclone.org/s3/#key-management-system-kms).

```

File: en/restore-from-s3.md [hunk 8]
```diff
@@ -23,6 +23,18 @@ Refer to [Prerequisites](restore-from-aws-s3-using-br.md#prerequisites-for-ad-ho
 
 ## Restoration process
 
+> **Note:**
+>
+> Due to the [issue](https://github.com/rclone/rclone/issues/1824) of `rclone`, if the backup data is stored in AWS S3 and the `AWS-KMS` encryption is enabled, you have to configure `spec.s3.options` as below in the yamls used in the following procedures, for more detail, refer to the [document](https://rclone.org/s3/#key-management-system-kms):
```
Comment (ran-huang) [inline en/restore-from-s3.md:28]:
```suggestion
> Due to the [issue](https://github.com/rclone/rclone/issues/1824) of `rclone`, if the backup data is stored in AWS S3 and the `AWS-KMS` encryption is enabled, you need to configure `spec.s3.options` in the YAML file in the following steps:
```

File: en/restore-from-s3.md [hunk 9]
```diff
@@ -23,6 +23,18 @@ Refer to [Prerequisites](restore-from-aws-s3-using-br.md#prerequisites-for-ad-ho
...
+>
+> ```yaml
+> spec:
+>   ...
+>   s3:
+>     ...
+>     options:
+>     - --ignore-checksum
+
```
Comment (ran-huang) [inline en/restore-from-s3.md:37]:
```suggestion
> ```
>
> For more details, refer to the [rclone | Key Management System](https://rclone.org/s3/#key-management-system-kms).

```

File: zh/backup-to-s3.md [hunk 10]
```diff
@@ -26,6 +26,19 @@ Ad-hoc 全量备份通过创建一个自定义的 `Backup` custom resource (CR)
...
+> * 由于 `rclone` 存在[问题](https://github.com/rclone/rclone/issues/1824)，如果使用 Amazon S3 存储备份，并且 Amazon S3 开启了 `AWS-KMS` 加密，需要在下面步骤中的 yaml 文件里配置 `spec.s3.options` 以保证备份成功，详细解释可以参考[文档](https://rclone.org/s3/#key-management-system-kms)：
+>
+>     ```yaml
+>     spec:
+>       ...
+>       s3:
+>         ...
+>         options:
+>         - --ignore-checksum
+
```
Comment (ran-huang) [inline zh/backup-to-s3.md:41]:
```suggestion
>    ```

```

File: zh/backup-to-s3.md [hunk 11]
```diff
@@ -225,6 +244,21 @@ Amazon S3 支持以下几种 `storageClass` 类型：
...
+>     ```yaml
+>     spec:
+>       ...
+>       backupTemplate:
+>         ...
+>         s3:
+>           ...
+>           options:
+>           - --ignore-checksum
+
```
Comment (ran-huang) [inline zh/backup-to-s3.md:261]:
```suggestion
>     ```

```

File: zh/restore-from-s3.md [hunk 12]
```diff
@@ -22,6 +22,18 @@ category: how-to
...
+> 由于 `rclone` 存在[问题](https://github.com/rclone/rclone/issues/1824)，如果使用 Amazon S3 存储备份，并且 Amazon S3 开启了 `AWS-KMS` 加密，需要在下面步骤中的 yaml 文件里配置 `spec.s3.options` 以保证备份恢复成功，详细解释可以参考[文档](https://rclone.org/s3/#key-management-system-kms)：
+>
+> ```yaml
+> spec:
+>   ...
+>   s3:
+>     ...
+>     options:
+>     - --ignore-checksum
+
```
Comment (ran-huang) [inline zh/restore-from-s3.md:36]:
```suggestion
> ```

```

**Comment by lilin90** [comment]
@DanielZhangQD I added two commits. PTAL

**Pull request merged**